src/ERC721Drop.sol

```diff
@@ -1267,6 +1272,38 @@ contract ERC721Drop is
         emit MintFeePayout(zoraFee, ZORA_MINT_FEE_RECIPIENT, success);
     }
 
+    function _requireCanMintQuantity(uint256 quantity) internal view {
+        if (quantity + _totalMinted() > config.editionSize) {
+            revert Mint_SoldOut();
+        }
+    }
+
+    function _handleSupplyRoyalty(uint256 mintAmount) internal returns (uint256 totalRoyaltyMints) {
```

Comment:
can this have a better name that describes what it's doing? `_handle` is a bit vague.  maybe `_mintSupplyRoyalty` ?

Reply:
`mintQuantity` would be better - amount to be sounds like an eth value.